.gitignore

```diff
@@ -24,3 +24,14 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+
+/config/credentials/development.key
+
+/config/credentials/test.key
+
+.env.development.local
```

Comment:
Why are there so many?  I'm more confused than a homeless man on house arrest 😫

Reply:
hahahah those are all files that **if** created the the `dot-env`  gem will use them. The difference between them it's on the precedence that the `dot-env`gem gives to each file. I'm just using the plain _.env_ file, but I added all those to the .gitignore in case somebody use some of them

Reply:
I would leave them out of gitignore until they exist. If someone adds them then they can add them here WDYT

Reply:
yeah ! No problem